examples/telegram-bot/TelegramBot.ts

```diff
@@ -1,14 +1,14 @@
 import dotenv from 'dotenv';
 import { Telegraf } from 'telegraf';
 import { ChatOpenAI } from '@langchain/openai';
-import { PolkadotAgentKit } from '../../../src/agent/index';
-import { PolkadotLangTools } from '../../../src/tools/index';
+import { PolkadotAgentKit } from '../../src/agent/index';
```

Comment:
```suggestion
import { PolkadotAgentKit } from '@/agent';
```

Can you please create path alias https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353. 
And we don't need to add `/index`.